openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -103,6 +103,20 @@ <h3 class="vc-title vc-fade vc-polite-only">Pursue a verified certificate</h3>
                 % endif
             </main>
             <aside class="course-sidebar layout-col layout-col-a">
+                % if has_goal_permission:
+                    <div class="section section-goals ${'' if cur_goal else 'hidden'}">
+                        <div class="current-goal">
+                            <span class="sr">${_("Set your course goal:")}</span>
```

Comment:
It is weird that there is extra context given for screen readers and not everybody.

Reply:
Was adding this because with only the screen reader you don't get the affordance of the pencil icon to know that this title is for setting and not just viewing your course goal.